packages/patternfly-4/react-integration/demo-app-ts/src/components/demos/StackDemo/StackDemo.tsx

```diff
@@ -0,0 +1,14 @@
+import React, { Component } from 'react';
+import { Stack, StackItem } from '@patternfly/react-core';
+
+export class StackDemo extends Component {
+  render() {
+    return (
+      <Stack gutter="sm">
```

Comment:
Would be nice to add an example with an alternative component ti verify it renders as expected.

Reply:
Ok, I added a badge component :)